azure-fluid-relay-preview-pr/articles/howtos/hosting.md

```diff
@@ -0,0 +1,258 @@
+---
+title: Hosting Fluid Example
+description: Detailed explanation about how Fluid examples can be hosted on Azure App Service and Static Web App
+---
+
+This article demonstrates how to deploy your Fluid examples to Azure App Service and Static Web Apps. [FluidHelloWorld](https://github.com/microsoft/FluidHelloWorld) repository is a Fluid example which contains a simple app that enables all connected clients to roll a dice and view the result. `FluidHelloWorld` is framework agnostic and provides sample code for rendering three different views: React, Vue and VanillaJS. Please pick the one that suits your development requirements.
+
+# Pre-requisites
+1. Add the following extensions to Visual Studio Code: 
+    - [Azure Account](https://marketplace.visualstudio.com/items?itemName=ms-vscode.azure-account): It is required to sign in to your `Azure Subscription`.
+    - [Azure App Service](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azureappservice): It is required for deploying app using `App Services`.
```

Comment:
There should be an indicator here that the dev can include one or the other, App Service or Static Web App. They don't need to install both